Source/Scene/Scene.js

```diff
@@ -1471,8 +1492,12 @@ define([
             executeCommand(skyAtmosphereCommand, scene, context, passState);
         }
 
+        if (defined(sunComputeCommand)) {
```

Comment:
I don't think we need the texture until the sun is visible so let's try to delay it.  It could help some apps in a small way.